### PR TITLE
General updates to the project

### PR DESCRIPTION
**Pull request by Makkkkus**
Reopened pull request because last one was not being looked at.

This is ready to merge.

### CODE REVIEW

**Comment by africalinguist** [comment]
check off let people use ports to connect (since that was just done in the recent merge by alice) and I'll merge it

File: src/minicraft/entity/mob/Creeper.java [hunk 1]
```diff
@@ -92,33 +93,33 @@ public void tick() {
 						Mob mob = (Mob) entity;
 						int distx = Math.abs(mob.x - x);
 						int disty = Math.abs(mob.y - y);
-						float distDiag = (float) Math.sqrt(distx * distx + disty * disty);
+						float distDiag = (float) Math.sqrt(distx^2 + disty^2);
```
Comment (ChristofferHolmesland) [inline src/minicraft/entity/mob/Creeper.java:96]: The `^` operator is a bitwise XOR. Undo this change.

File: src/minicraft/level/tile/TorchTile.java [hunk 2]
```diff
@@ -44,7 +43,7 @@ public void render(Screen screen, Level level, int x, int y) {
 	}
 	
 	public int getLightRadius(Level level, int x, int y) {
-		return 4;
+		return 5;
```
Comment (ChristofferHolmesland) [inline src/minicraft/level/tile/TorchTile.java:46]: Why?

File: src/minicraft/screen/TitleDisplay.java [hunk 3]
```diff
@@ -266,13 +246,10 @@ public void render(Screen screen) {
 		"Why is this blue?",
 		"Green is a nice color!",
 		"Red is my favorite color!",
-		"Y U NO BOAT!?",
```
Comment (ChristofferHolmesland) [inline src/minicraft/screen/TitleDisplay.java:269]: What's the reason for removing some of these?

**Review by ChristofferHolmesland — CHANGES_REQUESTED** [review]
Concerns:
- The code style is being changed without changing all of the instances of the old style. This means the code is a messy mix, e.g. only some for loops have a space between `for` and `(`. Suggestion: only change them if you change all of them.
- `xa` and `ya` are being changed to `xd`, `yd`, `xmov`, `ymov`. Why change them? And why are you changing them to two different names?
- The commit messages contain no useful information. There are too many changes in each commit, making it really hard to browse the history. Suggestion: Rewrite git history to split the changes into smaller commits with proper descriptions.

File: src/minicraft/screen/TitleDisplay.java (hunk 3 above)
Comment (Makkkkus) [reply src/minicraft/screen/TitleDisplay.java:269]: They're bad

File: src/minicraft/level/tile/TorchTile.java (hunk 2 above)
Comment (Makkkkus) [reply src/minicraft/level/tile/TorchTile.java:46]: Light is really weak

File: src/minicraft/entity/mob/Creeper.java (hunk 1 above)
Comment (Makkkkus) [reply src/minicraft/entity/mob/Creeper.java:96]: 1: I normally change them when I notice inconsistency. Don't want to change all of it at once, since they lead to messy commits.
2: See last commit; yd and xd are deltas, and xmov and ymov are distances. These two are different in some functions.
3: Fair enough

Comment (Makkkkus) [reply src/minicraft/entity/mob/Creeper.java:96]: > 
> 
> The `^` operator is a bitwise XOR. Undo this change.

Oh, my bad

**Comment by Makkkkus** [comment]
@ChristofferHolmesland completed changes requested
@africalinguist ready to merge

**Comment by chrisj42** [comment]
diff looks good.
